src/client/components/QRReaderComponent/QRReaderComponent.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './QRReaderComponent.styles.css';
+
+export default function QRReaderComponent() {
+  return (
+    <div className="qr-reader-background">
```

Comment:
Try to use semantic html

Reply:
Do you mean i should use section instead?